crates/bevy_math/src/rects/urect.rs

```diff
@@ -314,12 +314,12 @@ impl URect {
     pub fn inset(&self, inset: i32) -> Self {
         let mut r = Self {
             min: UVec2::new(
-                self.min.x.checked_add_signed(-inset).unwrap(),
-                self.min.y.checked_add_signed(-inset).unwrap(),
+                self.min.x.saturating_add_signed(-inset),
```

Comment:
Just need to delete the Panics doc above and should be good to merge I think. Thanks 👍